src/App.js

```diff
@@ -1,22 +1,39 @@
 import React from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  NavLink,
+} from 'react-router-dom';
 import './App.css';
 
-const App = () => (
-  <div className="App">
-    <HomePage />
-    <TabsPage />
-  </div>
-);
-
-const HomePage = () => <h1>Home page</h1>;
+import TabsPage from './components/TabsPage/TabsPage';
+import HomePage from './components/HomePage/HomePage';
 
 const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
-const TabsPage = () => <Tabs tabs={tabs} currentTabId="tab-1" />;
 
-const Tabs = () => <h2>Tabs</h2>;
+const App = () => (
+  <div className="app">
+    <Router>
+      <nav className="nav">
+        <NavLink className="nav-link" to="/">Home</NavLink>
```

Comment:
```suggestion
        <NavLink className="nav-link" exact to="/">Home</NavLink>
```

почитай что такое `activeClassName` у `NavLink`
(опционально)